### PR TITLE
[F2F-102] EU Driving Licence Expiry Screen

### PR DESCRIPTION
**Pull request by GDainton**
### Issue tracking

- [F2F-102](https://govukverify.atlassian.net/browse/F2F-XXX)

### Environment variables or secrets

- [ ] Documented in the README
- [x] Added screenshots to show the implementation is working
- [ ] Ran cfn-lint on any SAM templates

Screenshot 2023-01-12 at 13 52 23

Screenshot 2023-01-12 at 13 52 31

### Other considerations

[F2F-102]: https://govukverify.atlassian.net/browse/F2F-102?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by hemavana** [comment]
please add screenshots

**Comment by GDainton** [comment]
Screenshot 2023-01-12 at 13 52 31
Screenshot 2023-01-12 at 13 52 31

**Comment by GDainton** [comment]
Screenshot 2023-01-12 at 13 52 31
Screenshot 2023-01-12 at 13 52 23

File: src/views/cic/euPhotocardDlDetails.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    {% endcall %}
+
+{% endblock %}
+
+{# generate the specific footer items required for the PYI flows #}
+{% set footerNavItems = translate("govuk.footerNavItems") %}
+
+{% block footer %}
+    {{ govukFooter( footerNavItems ) }}
+{% endblock %}
```
Comment (hemavana) [inline src/views/cic/euPhotocardDlDetails.html:46]: new line to EOF

**Review by hemavana — DISMISSED** [review]
add new line to EOF for some files, then good to merge in

**Review by richa-misra — APPROVED** [review]

**Pull request merged**